### PR TITLE
Updated platform support with .NET 6

### PR DESCRIPTION
**Pull request by snakefoot**

### CODE REVIEW

**Comment by 304NotModified** [comment]
I really prefer using the official names. So we do .NET (Core) etc.

When people are searching on the web, I assume the also use the official names.

**Comment by snakefoot** [comment]
> I really prefer using the official names. So we do .NET (Core) etc.

Changed .NET CoreApp to .NET Core

File: index.html [hunk 1]
```diff
@@ -123,21 +123,15 @@ <h1 class="page-header" id="support">Support</h1>
...
+                <li>UWP (.NET Standard)</li>
                 <li>Mono 4</li>
-                <li>ASP.NET 4 (NLog.Web package)</li>
                 <li>ASP.NET Core (NLog.Web.AspNetCore package)</li>
-                <li>.NET Core (NLog.Extensions.Logging package)</li>
-                <li>.NET Standard 1.x - NLog 4.5</li>
-                <li>.NET Standard 2.x - NLog 4.5</li>
-                <li>UWP - NLog 4.5</li>   
-              </ul>
+                <li>ASP.NET Classic (NLog.Web package)</li>
```
Comment (304NotModified) [inline index.html:133]: Is that called "classic" nowadays?  Sounds like ASP classic ;)

Suggestion: ASP.NET or ASP.NET (non core)

Comment (304NotModified) [reply index.html:133]: Microsoft calls it also ASP.NET 4.x: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/choose-aspnet-framework?view=aspnetcore-2.1#aspnet-4x

Comment (snakefoot) [reply index.html]: Changed to ASP.NET (non core).

**Comment by snakefoot** [comment]
Ready for merge?

**Comment by 304NotModified** [comment]
Yes done :)

**Pull request merged**